test/unit/api/solver_black.h

```diff
@@ -744,12 +744,13 @@ void SolverBlack::testMkPi() { TS_ASSERT_THROWS_NOTHING(d_solver->mkPi()); }
 
 void SolverBlack::testMkReal()
```

Comment:
Create a test `testMkInteger` where you more or less duplicate `testMkReal` and convert all `mkReal` tests to `mkInteger`. Do *not* convert any `mkReal` test in `testMkInteger` but leave as is.
This applies to both `testMkReal` functions.

Reply:
Makes sense. Done